content/documentation/commands.md

```diff
@@ -0,0 +1,136 @@
++++
+title = "CLI Commands"
+weight = 21
++++
+
+Phel includes a series of commands out-of-the-box.
+
+```bash
+# To see an overview of all commands.
+vendor/bin/phel list
+```
+
+## Build the project
+
+```bash
+php phel build
```

Comment:
Currently the name is `compile`. We should not call it build until we released the change.

Reply:
Ah, right, in the latest 0.8 is still compile. Ok, I will change it to that one, unless till we release 0.9 with so many things so far :)